src/schema/annotation.yaml

```diff
@@ -249,7 +210,6 @@ slots:
       - biolink:strand
 
   direction:
-    domain: Reaction
```

Comment:
Does this slot need a new domain?

Reply:
[reaction](https://microbiomedata.github.io/nmdc-schema/direction/) isn't being used by any class now, so we could just deprecate it.

Reply:
Is there a reason not to deprecate it?

Reply:
no, there isn't any reason to avoid deprecating. see list of other removed elements below.